include/benchmark/benchmark.h

```diff
@@ -902,13 +889,14 @@ class LambdaBenchmark : public Benchmark {
 
 }  // namespace internal
 
-inline internal::Benchmark* RegisterBenchmark(const char* name,
-                                              internal::Function* fn) {
-  return internal::RegisterBenchmarkInternal(
-      ::new internal::FunctionBenchmark(name, fn));
+#ifdef BENCHMARK_HAS_CXX11
+
+namespace internal {
+// Take ownership of the pointer and register the benchmark. Return the
+// registered benchmark.
+Benchmark* RegisterBenchmarkInternal(Benchmark*);
```

Comment:
why is this now under HAS_CXX11? I don't think it was before.

Reply:
Because this is only needed if you use lambda benchmarks, function and fixture benchmarks are registered using the non-internal RegisterBenchmark().